### ISSUE TITLE
Timestep arg does not do anything

### ISSUE THREAD

**Issue by n00mkrad** [nihui/ifrnet-ncnn-vulkan#1]
Same issue as in rife-ncnn-vulkan.

Changing timestep argument does not have any effect.

**Comment by nihui** [comment]
Vimeo90K models can only do 0.5 timestamp interpolation.

GoPro models can do arbitrary timestamp interpolation.

**Comment by Hudaldadi** [comment]
it works - the GoPro models are not as impressive like Vimeo ones.

Thank sooooooo much for sharing Your great work *
It works fantastic on Mac OSX!

**Comment by n00mkrad** [comment]
> Vimeo90K models can only do 0.5 timestamp interpolation.
> 
> GoPro models can do arbitrary timestamp interpolation.

I know, but the `-s` argument does not work, even when using the GoPro models. `-s 0.25` will do 2x interpolation instead of 4x.

**Comment by Hudaldadi** [comment]
maybe the -s argument is only choosing the position in time and does not generate more frames.

**Comment by NeedsMoar** [comment]
No, the code is just broken.  When processing an entire directory, it always uses numframes / the count of files to control this.  You need to specify how many frames you want and it should work.  OTOH when only passing in single files only the timestep argument does anything.  As clear as the -tta flag that all the upscalers provide with no documentation which sometimes lowers quality.
